src/vue/backend/src/views/GuidedSetupConnectIDXBrokerAccount.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <GuidedSetupContentCard class="gs" cardTitle="Connect Your IDX Broker Account" :steps="steps" :relatedLinks="links">
+        <template v-slot:description>
+            <p>This step is optional. A sentence or two about why you should connect IMPress for IDX Broker to your IDX Broker account. Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+        </template>
+        <template v-slot:controls>
+            <APIKey/>
+        </template>
+    </GuidedSetupContentCard>
+</template>
+
+<script>
+import GuidedSetupContentCard from '@/templates/GuidedSetupContentCard.vue'
+import APIKey from '@/components/APIKey.vue'
+export default {
+    name: 'GuidedSetupConnectIDXBrokerAccount',
+    components: {
+        APIKey,
+        GuidedSetupContentCard
+    },
+    data () {
+        return {
+            steps: [
+                { name: '1. Connect', icon: 'link', total: 2, active: 1 },
```

Comment:
I can see this leading to a lot of duplicate work, try making a store module for the progress stepper to contain the base information, and we can update the active number using actions as we move through the flow